recommender/conf/routes

```diff
@@ -2,7 +2,7 @@
 # This file defines all application routes (Higher priority routes first)
 # ~~~~
 
-GET        /healthcheck                     controllers.Application.healthCheck
+GET        /healthcheck                             controllers.Application.healthCheck
 
-# GET         /recommendations/:userId        controllers.Application.recommendationsFromUserId(userId, webPublicationDate: Option[DateRangeFilter], tags: Option[QueryBoost], disableDateFilter: Option[Boolean], pageSize: Option[Int])
```

Comment:
is it expected that recommendationsFromUserId disapears? I guess it's not used ?